Libraries/CameraRoll/CameraRoll.js

```diff
@@ -58,6 +58,11 @@ var getPhotosParamChecker = createStrictShapeTypeChecker({
    * titles.
    */
   groupName: ReactPropTypes.string,
+
+  /**
+  * Specifies filter on asset type, like 'all', 'photos' or 'videos'
+  */
+  assetType: ReactPropTypes.string,
```

Comment:
can you make it ReactPropTypes.oneOf(['all', 'photos', 'videos']) instead of writing it in the comments. The comment can say that `all` is the defaul